index.bs

```diff
@@ -215,12 +214,21 @@ is defined as follows:
 * If |N| is an {{Element}} which generates one or more <a>boxes</a>, the
     starting point of |N| in |C| is the two-dimensional offset in <a>pixel
     units</a> from the origin of |C| to the <a>flow-relative</a> starting corner
-    of the first <a>fragment</a> of the <a>principal box</a> of |N|.
+    of the first <a>fragment</a> of the <a>principal box</a> of |N|, calculated
+    as if every <a>transformed element</a> in the <a>node document</a> of |N|
```

Comment:
Perhaps this should be a bit clearer, as an element in a document can be a bit ambiguous. Does this mean every element whose node document is that document? It could also mean every element whose root is that document, but I suspect it's the former in this case (i.e. no special treatment for shadow trees?). Ditto for the equivalent text below.

Reply:
How about removing the reference to node document entirely?  Ignore transforms for all elements in the universe.

Reply:
Sounds good to me.